bears/general/AnnotationBear.py

```diff
@@ -290,6 +295,17 @@ def get_singleline_comment(file, filename, text, comment, position):
                 end_position)
 
 
+def get_if_present(language, attribute, default={}):
```

Comment:
`def _get_lang_attr` and invoke it with `_get_lang_attr(.., ..., default=..)` which automatically implies that default will apply if the attr is not present.

Reply:
uhm why don't you just use `getattr(language, attribute, default)` in the first place? No need to define this function

Reply:
https://docs.python.org/3/library/functions.html#getattr